app/controllers/IncomeController.scala

```diff
@@ -110,34 +125,49 @@ class IncomeController @Inject()(personService: PersonService,
           journeyCacheService.collectedValues(Seq(UpdateIncome_PayToDateKey, UpdateIncome_IdKey, UpdateIncome_NameKey), Seq(UpdateIncome_DateKey)) flatMap tupled {
             (mandatorySeq, optionalSeq) => {
               val date = optionalSeq.head.map(date => LocalDate.parse(date))
-              EditIncomeForm.bind(mandatorySeq(2), BigDecimal(mandatorySeq.head), date).fold(
-                formWithErrors => {
+              val employerName = mandatorySeq(2)
+
+              EditIncomeForm.bind(employerName, BigDecimal(mandatorySeq.head), date).fold(
+                (formWithErrors: Form[EditIncomeForm]) => {
                   val webChat = true
```

Comment:
Do we need this line?

Reply:
Yeah looks like it turns on webchat when there's something gone wrong when editing pensions